vegafusion-rt-datafusion/src/data/tasks.rs

```diff
@@ -113,7 +113,7 @@ impl TaskCall for DataUrlTask {
                     VegaFusionDataset::Table { table, .. } => table.to_sql_dataframe().await?,
                     VegaFusionDataset::SqlDataFrame(sql_df) => sql_df.clone(),
                 };
-                let sql_df = process_datetimes(&parse, sql_df, &config.tz_config)?;
+                let sql_df = process_datetimes(&parse, sql_df, &config.tz_config).await?;
```

Comment:
The DataFusion update made a few core functions async, so that's propagated through our code base a bit more as well.